eng/Version.Details.xml

```diff
@@ -119,11 +119,11 @@
     <!-- Necessary for source-build. This allows Microsoft.Extensions.ObjectPool and System.Collections.Immutable packages
          to be retrieved from live source-build and their content consumed by packages produced by razor.
          Without these entries, due to PVP flow work, SBRP packages would be consumed causing runtime issue. -->
-    <Dependency Name="Microsoft.Extensions.ObjectPool" Version="6.0.0">
+    <Dependency Name="Microsoft.Extensions.ObjectPool" Version="8.0.0">
       <Uri>https://github.com/dotnet/aspnetcore</Uri>
       <Sha>ae1a6cbe225b99c0bf38b7e31bf60cb653b73a52</Sha>
```

Comment:
I have no idea how this works, but its very surprising that we can bump a version without changing the sha.

Reply:
Great point! I'm not exactly sure where to get the correct SHA. Maybe @dotnet/source-build-internal knows, since they need to approve this change anyway.

Reply:
The easiest is to just look at the release tag of the repo: https://github.com/dotnet/aspnetcore/tree/v8.0.0. In this case, it is point to commit `3f1acb59718cadf111a0a796681e3d3509bb3381`.